graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/shared/CodeGeneratorUtils.kt

```diff
@@ -30,7 +30,7 @@ object CodeGeneratorUtils {
      * */
     fun camelCaseToSnakeCase(input: String, case: Case = Case.LOWERCASE): String {
         val parts = splitByCharacterTypeCamelCase(input)
-        return parts.joinToString(separator = "_") {
+        return parts.filter { c -> c != "_" }.joinToString(separator = "_") {
```

Comment:
it might have been cleaner/maybe faster to do it in the `splitByCharacterTypeCamelCase()` method to not iterate twice